members/FMNS420b9c88.yaml

```diff
@@ -43,7 +43,7 @@ contact_form:
         required: true
       - name: subtopic
         selector: input#edit-submitted-message-details-sub-topic
-        value: "$SUBTOPIC1"
+        value: "$TOPIC"
```

Comment:
are the main topics going to correctly map to the subtopics?

Reply:
good question. All the tests have passed, but without looking at what the final successful message looks like in the form I can't say for sure. What about using the $SUBJECT variable instead here? I think that'd be a better way to make sure the subtopic requirement is fulfilled

Reply:
1. `#edit-submitted-message-details-topic` should be in a `select` step with a value of `$TOPIC`
2. `#edit-submitted-message-details-sub-topic` should be in a `fill-in` step with a value of `$SUBJECT`

Reply:
Perfect! I'll do the edit and commit them, thanks